### ISSUE TITLE
Composer install started failing yesterday "Could not authenticate against github.com"

### ISSUE THREAD

**Issue by phadaphunk** [composer/composer#11662]
Anyone experiencing failing composer install with issues like `  Could not authenticate against github.com  `

Exactly like it happened here https://github.com/composer/composer/issues/8710

 Tried pretty much everything proposed in that thread to no avail our installs have been failing randomly for more than 12 hours now. Some will pass randomly but most fail with the same error getting a [401] from github at a random point in the install process. Tried new github tokens but still have the same issue.

**Comment by Seldaek** [comment]
Maybe you could share more details, verbose output etc,.. That might give more of a clue about what is going on.

**Comment by phadaphunk** [comment]
@Seldaek The issue was on Github's side, they rolled out a fix.

**Comment by dpow-hrs** [comment]
Yes we are also getting this issue, only started today. As already mentioned last time this happened we had to wait for github to fix a mistake they'd made in their rate limiting.
Comment from GitHub here: https://github.com/composer/composer/issues/8710#issuecomment-604378533

**Comment by Seldaek** [comment]
You have to let them know though, just waiting may not result in anything being fixed.
